PWD_Vault/PWD_Vault.py

```diff
@@ -91,93 +96,99 @@ def login_screen():
-            sure = input('> ')
-            if sure == 'y':
-                with open('data.csv', 'w') as slaughter:
-                    slaughter.truncate()
-                    print('The data will never be found again!')
-                    input("Hit Enter to return to the main menu\n> ")
-                    main_menu()
+                main_menu()
+
+            case '2':
+                print('\n')
+                readc()
+                print('\n')
+                input("Hit Enter to return to the main menu\n> ")
+                main_menu()
 
-            elif sure == 'n':
-                print('No Problem')
+            case '3':
+                print('\n')
+                searchc()
+                print('\n')
+                input("Hit Enter to return to the main menu\n> ")
                 main_menu()
-            else:
-                print('\nINVALID OPTION')
+
+            case '4':
+                print('\n')
+                deletec()
                 input("Hit Enter to return to the main menu\n> ")
                 main_menu()
+
+            case 'h':
+                print('\n')
+                with open('PWD_Vault/PWD_Vault/help.txt', 'r') as manual:
+                    reader = manual.read()
+                    print(reader)
+                    input("Hit Enter to return to the main menu\n> ")
+                print('\n')
+                main_menu()
+
+            case 'e':
+                print('Are you sure you that you want to erase the data?(y/n) ')
+                sure = input('> ').lower()
+                if sure == 'y':
+                    with open('data.csv', 'w') as slaughter:
+                        slaughter.truncate()
+                        print('The data will never be found again!')
+                        input("Hit Enter to return to the main menu\n> ")
+                        main_menu()
+
+                elif sure == 'n':
+                    print('No Problem')
+                    main_menu()
+                else:
+                    print('\nINVALID OPTION')
+                    input("Hit Enter to return to the main menu\n> ")
+                    main_menu()
         
-        elif option == 'q':
-            print('Encrypting Data...')
-            encrypt(key)
-            print('Exiting...')
-            quit()
+            case 'q':
+                print('Encrypting Data...')
+                encrypt(key)
+                print('Exiting...')
+                quit()
         
-        elif option == 'banner':
-            banner_p()
-            main_menu()
-        else:
-            print('INVALID OPTION!')
-            main_menu()
+            case 'banner':
+                banner_p()
+                main_menu()
+                
+            case _:
+                print('INVALID OPTION!')
+                main_menu()
 
     except KeyboardInterrupt:
         print("\n!!!!!Use 'q' to exit!!!!")

```

Comment:
Can you add this in the login menu too. This exception is currently in only in the main menu after you login.